_posts/2019-01-20-why-we-run-our-blog.md

```diff
@@ -0,0 +1,179 @@
+out: [AirBnB](https://medium.com/airbnb-engineering), [Harrys](https://medium.com/harrys-engineering),
+[Pintrest](https://medium.com/@Pinterest_Engineering), [Vimeo](https://medium.com/vimeo-engineering-blog) or
+[Lyft](https://eng.lyft.com)'s pages.
+
+Not memorable at all, because you can't really work with the design constraints to do anything creative other than
+a banner image.
+
+These constrains become worse on a post page, where you completely lose any sense of connection with the company,
+and the team the moment someone scrolls an inch until the footer. Your team's writing becomes just "a medium post"
+at that point. You've got limited options for attaching images, and no ability to use HTML/JS to showcase
+[problems][rn] [interactively][ar].
+
+These aren't blockers in any way, most of our posts don't do that - but the constraints mean you will never think
+to try and explain something outside of that sandbox.
+
+Then at the bottom of your post, readers are redirected to other posts from other teams. For example, I opened a
+post on Vimeo's announcement of [Psalm v3][psalm] (a cool looking PHP dev tool), there were three recommended
+posts: one was about missiles being fired in Syria, another was a beginners guide to PHP and then a third was how
+to set up Docker to work with a PHP framework. They're not that related, maybe they all have the keyword of PHP
+behind the scenes?
+
+{% include epic_img.html url="/images/hosting-our-own-blog/3.jpg" title="Artsy as a slideshow" %}
+
+Writing takes a lot of time, and provides so much value. It should be presented as [quality worthy of art][qwoa].
+By using to a generic platform for your writing, you're trading that simplicity for building your team's online
+presence.
+
+## Breaking the Sandbox
+
+We've grown to need to showcase quite a few different types of posts
+
+- Small posts that with only a few paragraphs
+- Long-form posts that take forever to read
+- Sequential posts, in the form of a series
+- YouTube embed posts
+- Announcements
+- Guest Posts
+
+None of these need to be treated the same, and since we created the blog, we've added:
+
+- Category pages - [GraphQL][graphql]
+- Author pages - [mine][author]
+- Site series - [React Native at Artsy][rnaa]
+- [4 separate post layouts][layouts]
+- Multi-author posts - [Pair Programming][pp]
+- [GitHub Issue Powered Comments][ghc]
+
+When we've wanted to add a new feature to the blog to fit a particular post, we added the feature. This gave us the
+chance to not constrain ourselves in ideas. For example, we've explored [building a podcast][podcast] into our blog
+treating it as a first class feature in ways that no-one would ever build if it was a platform. Or we're interested
+in making a way to highlight useful links for the
+
+## Reviews
+
+Because our [blog posts][_posts] are markdown in a GitHub repo, we don't treat a review for a blog post any
+different than a normal pull request for code.
+
+## Migrating
```

Comment:
Maybe this section should even mention that migrating between any of the static page generators is possible. e.g. exporting all my posts from SquareSpace was a pain, and I had to manually update a lot of things to get it to work. By having the markdown and CSS files you're pretty independent